lerna.json

```diff
@@ -6,7 +6,9 @@
     "packages/test-util",
     "packages/util",
     "test-packages/integration-tests",
+    "test-packages/e2e-tests",
```

Comment:
Just to be 100% sure. I see that the two packages are marked as `private` is this sufficient that they are not getting published?

Reply:
I think yes, see the [doc](https://docs.npmjs.com/files/package.json) about the `package.json`.